com.unity.uiwidgets/Runtime/scheduler2/binding.cs

```diff
@@ -533,76 +553,86 @@ public class SchedulerBinding : PaintingBinding {
 
         public void handleBeginFrame(TimeSpan? rawTimeStamp) {
             Timeline.startSync("Frame");
-
+            D.setDebugLog(_debugLog: UIWidgetsPanelWrapper.current.ShowDebugLog);
             _firstRawTimeStampInEpoch = _firstRawTimeStampInEpoch ?? rawTimeStamp;
-            _currentFrameTimeStamp = _adjustForEpoch(rawTimeStamp ?? _lastRawTimeStamp);
+            _currentFrameTimeStamp = _adjustForEpoch(rawTimeStamp ?? currentSystemFrameTimeStamp);
```

Comment:
I know currentSystemFrameTimeStamp == _lastRawTimeStamp, but why do you decide to make this useless change?

Reply:
I guess it's the result of code cleanup. I' ll fix it .